WalletWasabi.Fluent/ViewModels/Settings/BitcoinTabSettingsViewModel.cs

```diff
@@ -115,5 +117,22 @@ protected override void EditConfigOnSave(Config config)
 				BitcoinP2PEndPoint = config.GetBitcoinP2pEndPoint().ToString(defaultPort: -1);
 			}
 		}
+
+		public static string GetNodeVersion()
+		{
+			var version = Constants.BitcoinCoreVersion;
+			try
+			{
+				var node = Node.Connect(Services.Config.Network, Services.Config.GetBitcoinP2pEndPoint().ToString(defaultPort: -1));
```

Comment:
Prefer non-blocking calls. In this case `Node.ConnectAsync`. Also remember `Node` is disposable and we dispose them by using `using var node = XXXX`

Reply:
the `ToString(defaultPortt: -1)` is unnecessary. Remove it, please.

Reply:
Done in https://github.com/zkSNACKs/WalletWasabi/pull/6942/commits/af03334456ed50f28b622d47f0d15d0e8c3a428f